app/presenters/publishing_api/worldwide_organisation_presenter.rb

```diff
@@ -36,5 +36,16 @@ def links
     def description
       item.summary
     end
+
+    def social_media_links
+      item.social_media_accounts.map do |account|
```

Comment:
I think we can do the same here with `.where(locale: I18n.locale)`. I realise it's less important because these pages are fully rendered by Whitehall still, but I think worth being consistent.

Reply:
Yes, I'd missed that one!